static/style/blue/base.css

```diff
@@ -1039,8 +1039,7 @@ pre {
   margin-bottom: 1rem;
   background-color: transparent; }
   .table th, .ui-table-striped th,
-  .table td,
-  .ui-table-striped td {
+  .table td, .ui-table-striped td {
```

Comment:
this whole file has been added to the commit accidentally I think

Reply:
Ugh how did that get there :(